### PR TITLE
Changes for MicroProfile 3.2-RC1 release.

### PR DESCRIPTION
**Pull request by jclingan**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -109,7 +109,7 @@
         <config-version>1.3</config-version>
         <ft-version>2.0</ft-version>
         <health-version>2.1</health-version>
-        <metrics-version>2.1.0</metrics-version>
+        <metrics-version>2.2-RC1</metrics-version>
```
Comment (kwsutter) [inline pom.xml]: Now that Metrics 2.2 released, then this should be 2.2

**Comment by jclingan** [comment]
I was just thinking this. Yeah, I agree.

File: pom.xml (hunk 1 above)
Comment (jclingan) [reply pom.xml]: Whups, email reply didn't apply to this thread. Yeah, I agree.

Comment (ederks85) [reply pom.xml]: So, skip the RC and go straight to final?

Comment (ederks85) [reply pom.xml]: Agreed

**Pull request merged**